darts/models/forecasting/arima.py

```diff
@@ -91,6 +91,7 @@ def _predict(
         historic_future_covariates: Optional[TimeSeries] = None,
         future_covariates: Optional[TimeSeries] = None,
         num_samples: int = 1,
+        verbose: bool = None,
```

Comment:
I find it awkward to initialize a `bool` to `None`. Couldn't we set it to `False`?

Reply:
I set it to `None` because `_setup_trainer` was doing so, in order to differentiate the cases where the user actually provide a value. I somehow also wanted to indicate that this argument is actually not used in 90% of these `predict` methods since the verbosity in handled elsewhere. I would not mind use `False` as a default value.

Reply:
I think False is clearer; IMO it'd actually be helpful for users to know what is the behavior if this one is not provided. WDYT?

Reply:
I set it to `None` for TorchForecastingModels to know when the user actively changed the verbosity

Reason was that user can activate/deactivate the verbosity at model creation (pl_trainer_kwargs such as `enable_progress_bar`, ...).
If we didn't use `None`, then the prameters from model creation would just be bypassed.

Looking at it now, I think it's probably okay to actually ignore these params from model creation, and only rely on a boolean `verbose` for TorchForecastingModels. Let me know what you think @hrzn and @madtoinou

Reply:
We would have to do it for both.
As otherwise, if the user wants actually no verbosity, then he would have to recreate the model with deactivated `enable_progress_bar`, ... (by default it is activated)

Reply:
right, yes, makes sense

Reply:
Thanks to both of you for the feedback, I'll change all the default value to False. @dennisbader, are you taking care of the PL-related verbosity argument overwriting or should I do it before you merge your change into this branch?